src/script/script.cpp

```diff
@@ -130,6 +130,9 @@ const char* GetOpName(opcodetype opcode)
     case OP_DETERMINISTICRANDOM    : return "OP_DETERMINISTICRANDOM";
     case OP_CHECKSIGFROMSTACK      : return "OP_CHECKSIGFROMSTACK";
     case OP_CHECKSIGFROMSTACKVERIFY: return "OP_CHECKSIGFROMSTACKVERIFY";
+    case OP_SHA256_INIT            : return "OP_SHA256_INIT";
```

Comment:
Might as well call it `OP_SHA256_INITIALIZE`.

Reply:
sounds better to me, rhymes with `OP_SHA256_FINALIZE`